src/llamafactory/api/chat.py

```diff
@@ -14,9 +14,11 @@
 
 import base64
 import io
+import re
 import json
 import os
 import uuid
+import numpy as np
```

Comment:
Please run `make style && make quality` before submitting code

Reply:
Sry about that this is changed online so....

Reply:
My bad.